ext/styledmap/eu.esdihumboldt.hale.doc.user.views.styledmap/html/customtile_detail.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Map configuration via Custom Tile URL</title>
+</head>
+<body>
+	<h1>Custom Tile Server Configuration</h1>
+	<p>
+		By configuring custom tile map servers, you can use any map based on
+		the web tiling scheme in <i>HALE</i>'s map view.
+	</p>
+	<p>
+		To work with <i>Custom Tile</i> maps, first open the Map view, go to
+		the Window menu, select <i>Show View</i> and pick <i>Map</i>.
+	</p>
+
+	<div>
+		<img src="../images/map_perspective_toolbar.png"
+			title="Map perspective tool menu" class="fit"> <img
+			src="../images/map_perspective_open.png"
+			title="Map perspective tool menu" class="fit"
+			style="margin-left: 20px">
+	</div>
+
+	<p>By default, hale studio uses the Stamen Terrain map in the Map
+		view. The Stamen Terrain map is created by Stamen Design and provided
+		under CC BY 3.0. It contains data by OpenStreetMap, licensed under CC
+		BY SA.</p>
```

Comment:
As we have the possibility here, we should add the proper links as well. For the map, the company, OpenStreetMap and the licenses. See the [Stamen map page](http://maps.stamen.com/#terrain/12/37.7706/-122.3782) for the complete Attribution example with links.